tests/data_context/test_get_data_context.py

```
@@ -0,0 +1,230 @@
+        stores={
+            "new_expectations_store": {"class_name": "ExpectationsStore"},
+            "new_evaluation_parameter_store": {
+                "class_name": "EvaluationParameterStore"
+            },
+            "new_validation_result_store": {"class_name": "ValidationsStore"},
+        },
+        validations_store_name="new_validation_result_store",
+        data_docs_sites={},
+        validation_operators={},
+    )
+    context = gx.get_context(project_config=config, context_root_dir=context_path)
+    assert isinstance(context, BaseDataContext)
+    assert context.expectations_store_name == "new_expectations_store"
+
+
+@pytest.mark.cloud
+@mock.patch("requests.get")
+@mock.patch.dict(
+    os.environ,
+    {
+        "GE_CLOUD_BASE_URL": "http://hello.com",
+        "GE_CLOUD_ORGANIZATION_ID": "1",
+        "GE_CLOUD_ACCESS_TOKEN": "i_am_a_token",
+    },
+    clear=True,
+)
+def test_cloud_config_in_env_returns_cloud_context(
+    mock_request,
+    request_headers,
+):
+    mock_request.return_value.status_code = 200
+    try:
+        assert isinstance(
+            gx.get_context(),
+            CloudDataContext,
+        )
+    except:  # Not concerned with constructor output (only evaluating interaction with requests during __init__)
+        pass
+
+
+@pytest.mark.cloud
+@mock.patch("requests.get")
+@mock.patch.dict(
+    os.environ,
+    {
+        "GE_CLOUD_BASE_URL": "http://hello.com",
+    },
+    clear=True,
+)
+def test_most_cloud_config_in_param_base_url_in_env_returns_cloud_context(
+    mock_request,
+    request_headers,
+):
+    mock_request.return_value.status_code = 200
+    try:
+        assert isinstance(
+            gx.get_context(
+                ge_cloud_organization_id="1", ge_cloud_access_token="i_am_a_token"
+            ),
+            CloudDataContext,
+        )
+    except:  # Not concerned with constructor output (only evaluating interaction with requests during __init__)
+        pass
```

I'm confused about what's actually being tested here.
If you are using a blanket `try: except:` why even have the assert statement?

The comment next to `except` makes me think that maybe a `spy` would be a more appropriate test double for this test.
https://pytest-mock.readthedocs.io/en/latest/usage.html#spy

---

The PR has been updated to `mock.patch` the `retrieve_data_context_config_from_ge_cloud()` method instead, which negates the need for the `try:except`, which I believe was masking an error. Thank you for calling this out. It helped reveal a hole in the test.